tests/loadtests/geom/cube.h

```diff
@@ -3,70 +3,47 @@
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
Either don't format this file or fix the formatting for it so the vertices remain separated in the cube face - the formatting has made the comments nonsensical plus it is much harder to see the vertices when they are all run together.

Reply:
There is no way to exclude individual files from formatting. That's why we had to move everything external that is not expected to be formatted by us into a separate subdirectory where we disabled formatting.

You can exclude lines of code from being formatted (or entire files) by surrounding them with:

```cpp
// clang-format off
...
// clang-format on
```

There are examples of this in the generated files.